rdmo/projects/apps.py

```diff
@@ -7,4 +8,5 @@ class ProjectsConfig(AppConfig):
     verbose_name = _('Projects')
 
     def ready(self):
-        from . import rules
+        if settings.PROJECT_REMOVE_VIEWS:
+            from . import handlers
```

Comment:
I moved the implementation into the `projects` app. I think this makes sense as you suggested. Furthermore, I also moved it from `signals.py` to `handlers.py`. However, I am not happy with the linking into the `projects/apps.py` due to the configuration flag.